test/controllers/user_controller_test.exs

```diff
@@ -108,6 +131,15 @@ defmodule CoursePlanner.UserControllerTest do
     assert html_response(conn, 403)
   end
 
+  test "shows the user himself" do
+    user = insert(:user)
+    user_conn = Phoenix.ConnTest.build_conn()
+    |> assign(:current_user, user)
```

Comment:
you don't need to duplicate this part. just use "login_as" and it will give you a "conn" object
then the user already is assigned to it. you can access it via "conn.assigns.current_user"